content/en/docs/concepts/containers/images.md

```diff
@@ -121,8 +121,8 @@ example, run these on your desktop/laptop:
    1. Run `docker login [server]` for each set of credentials you want to use.  This updates `$HOME/.docker/config.json` on your PC.
    1. View `$HOME/.docker/config.json` in an editor to ensure it contains just the credentials you want to use.
    1. Get a list of your nodes; for example:
-      - if you want the names: `nodes=$( kubectl get nodes -o jsonpath='{range.items[*].metadata}{.name} {end}' )`
-      - if you want to get the IP addresses: `nodes=$( kubectl get nodes -o jsonpath='{range .items[*].status.addresses[?(@.type=="ExternalIP")]}{.address} {end}' )`
+      - if you want the names: `nodes=($( kubectl get nodes -o jsonpath='{range.items[*].metadata}{.name} {end}' ))`
+      - if you want to get the IP addresses: `nodes=($( kubectl get nodes -o jsonpath='{range .items[*].status.addresses[?(@.type=="ExternalIP")]}{.address} {end}' ))`
```

Comment:
Does this rely on the reader having a particular piece of software? What are the requirements?

Reply:
This converts the output from a string to a [bash array](https://tldp.org/LDP/Bash-Beginners-Guide/html/sect_10_02.html). Don't think there are any extra dependencies I'm aware of :thinking:.

Reply:
This won't work if the nodes only have internal IPs.

Reply:
@tengqm  Yes, you are correct, and probably should also be addressed. For this issue, I prefer to stick to the original intention of the PR of converting `nodes` to an array.

Reply:
I'd prefer to either:
 - not have the dependency on bash (eg write code that runs in a POSIX-compatible shell)
 - use tabs to provide solutions for different shells

If using tabs, let's also consider covering PowerShell. We try to be somewhat technology-agnostic, especially where there is lots of diversity in the ecosystem - like people's choice of shell.

Reply:
@sftim it makes sense. Looks like I have under looked at other shell options, lets close this PR since it's currently already POSIX-compatible.